src/System.Runtime.WindowsRuntime.UI.Xaml/src/System/Windows/Media/Animation/KeyTime.cs

```diff
@@ -61,12 +61,7 @@ public bool Equals(KeyTime value)
 
         public override bool Equals(object value)
         {
-            if (value == null || !(value is KeyTime))
-            {
-                return false;
-            }
-
-            return this == (KeyTime)value;
+            return value is KeyTime && (KeyTime)value;
```

Comment:
Well, the only thing more embarrassing than having a heavy-fingered moment caught by CI is not having it caught by CI.